apps/src/templates/currentUserRedux.js

```diff
@@ -188,11 +188,7 @@ export default function currentUser(state = initialState, action) {
     );
     // Calling Statsig separately to emphasize different user integrations
     // and because dual reporting is aspirationally temporary (March 2024)
-    statsigReporter.setUserProperties(
-      id,
-      user_type,
-      experiments.getEnabledExperiments()
```

Comment:
not a big deal but curious why the enabled experiments was removed?

Reply:
To be honest my initial thought was that we wouldn't be using the experiments framework alongside statsig's AB testing but I realize I'm not clear yet on how we will approach that